algorithms/kernel/service_heap.h

```diff
@@ -106,6 +106,38 @@ DAAL_FORCEINLINE void sortMaxHeap(RandomAccessIterator first, RandomAccessIterat
     }
 }
 
+/*
+Cut from kdtree_knn_classification_predict_dense_default_batch.i to provide more consistent access
```

Comment:
What does it mean?

Reply:
It means that some code was moved into ```service_heap.h```. All service functions related to heap should be placed there, right?